crux-test/test/crux/query_test.clj

```diff
@@ -2683,3 +2683,18 @@
                                            snapshot-hit-ns)))))
                          (repeatedly n))]
         (t/is (>= (/ (reduce + factors) n) acceptable-snapshot-speedup))))))
+
+(t/deftest test-greater-than-range-predicate-545
+  (t/is (empty?
+         (api/q (api/db *api*)
+                '{:find [offset]
+                  :where [[e :offset offset]
+                          [(> offset -9223372036854775808)]] ;; Long/MAX_VALUE
+                  :limit 1})))
+
+  (t/is (empty?
```

Comment:
separate test for this one? doesn't fit under `test-greater-than-range-predicate`?